webcomponents/tool/src/main/frontend/js/calendar/sakai-calendar.js

```diff
@@ -110,20 +110,32 @@ export class SakaiCalendar extends LionCalendar {
   get userId() { return this._userId; }
 
   shouldUpdate(changed) {
-    return this.events && super.shouldUpdate(changed);
+    return super.shouldUpdate(changed);
   }
 
   loadData() {
 
     const url = this.siteId
-      ? `/api/sites/${this.siteId}/calendar` : `/api/users/${this.userId}/calendar`;
+      ? `/api/sites/${this.siteId}/calendar` : `/api/users/current/calendar`;
```

Comment:
Are we considering using the 'current' pattern for sites too, as @profmikegreene suggested on slack?

Reply:
Not sure if we can. Current site doesn't get set up unless you come through portal.